lib/tests/formulae.rb

```diff
@@ -442,7 +442,7 @@ def formula!(formula_name, args:)
         audit_args << "--except=stable_version" if args.skip_stable_version_audit?
         audit_args << "--except=revision" if args.skip_revision_audit?
         if new_formula
-          audit_args << "--new"
+          audit_args << "--new" unless args.skip_repository_audit?
```

Comment:
We probably don't want to skip `--new` entirely because `--new` implies `--strict` and `--online` which include a bunch of other audits too (e.g., "Formulae in homebrew/core must specify a license.").

To really skip the repository audits, I suggest adding `--except=github_repository,gitlab_repository,bitbucket_repository` to the audit args instead. They should cover this category of new-formula audit failures we usually see in `homebrew/core`. See:

- https://github.com/Homebrew/brew/blob/567386a13ead16e510d228e93b0d43c1677e9d0b/Library/Homebrew/formula_auditor.rb#L629-L658
- https://github.com/Homebrew/brew/blob/567386a13ead16e510d228e93b0d43c1677e9d0b/Library/Homebrew/utils/shared_audits.rb#L105-L166

Reply:
Good call. I only saw the ones in `shared_audits` on first look. I've pushed the change.